test/dashboard-e2e/api/api-helpers.js

```diff
@@ -1,17 +1,26 @@
 import superagent from 'superagent';
 
 export class ApiHelpers {
-    constructor(apiUrl) {
+    constructor(apiUrl, cloudContext='', bearerToken='') {
```

Comment:
There is no need to default them to empty strings
```suggestion
    constructor(apiUrl, cloudContext, bearerToken) {
```